src/tileset.rs

```diff
@@ -24,9 +24,10 @@ pub struct Tileset {
     pub spacing: u32,
     pub margin: u32,
     pub tilecount: Option<u32>,
+    pub columns: u32,
     /// The Tiled spec says that a tileset can have mutliple images so a `Vec`
     /// is used. Usually you will only use one.
-    pub images: Vec<Image>,
+    pub image: Option<Image>,
```

Comment:
Please update the docs attached to the member

Reply:
My bad. Updated.